module-2/backend/week_two.md

```diff
@@ -26,11 +36,33 @@ end
 
 Now how would you find the owner of the team with an id of 4?
 
+`Team.find(4).owner`
+
 5. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+
+Teacher has many Students, a Student belongs to a Teacher
+
+```table students
+|      id    |
+|     name   |
+
+table teachers
+|     id     |
+|    name    |
+| student_id |```
+
 6. Define foreign key, primary key, and schema.
+
+A primary key is the `id` of a table. It is the key that is inserted to an associated table. The key referenced in the associated table is the `foreign key`. The schema is a blueprint of the database and all of your migrations.
+
 7. Describe the relationship between a foreign key on one table and a primary key on another table.
+
+A foreign key is the has_many part of an association, while a primary key is the id being inserted as the foreign key.
```

Comment:
a foreign key on one table will refer to the primary key of the table it is references